apps/publish/enqueue/enqueue_service.py

```diff
@@ -320,15 +320,20 @@ def queue_transmission(self, doc, subscribers, subscriber_codes={}):
                                                           subscriber,
                                                           subscriber_codes.get(subscriber[config.ID_FIELD]))
 
-                        for pub_seq_num, formatted_doc in formatted_docs:
-                            publish_queue_item = dict()
+                        for idx, publish_data in enumerate(formatted_docs):
```

Comment:
to be able to return a dictionary, like here: https://github.com/superdesk/superdesk/pull/2207/files#diff-5b639099f2d4d0f4d8acad9623b95628R44